packages/wonder-blocks-form/src/components/labeled-text-field.stories.js

```diff
@@ -37,6 +37,49 @@ export const Text: StoryComponentType = () => {
     );
 };
 
+export const RequiredWithDefaultText: StoryComponentType = () => {
+    const [value, setValue] = React.useState("");
+
+    const handleKeyDown = (event: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
+
+    return (
+        <LabeledTextField
+            label="Name"
+            description="Please enter your name"
+            value={value}
+            onChange={(newValue) => setValue(newValue)}
+            onKeyDown={handleKeyDown}
+            required={true}
+        />
+    );
+};
+
+export const RequiredWithSpecifiedText: StoryComponentType = () => {
```

Comment:
I definitively think that we don't need/want to send this story to Chromatic for visual regression, specially because this visual behavior is already tested in the previous story.

```
export const RequiredWithSpecifiedText: =  () => {
}
...

RequiredWithSpecifiedText.parameters = {
    chromatic: {
        // We have screenshots of other stories that cover this case.
        disableSnapshot: true,
    },
};
```